legal-api/src/legal_api/core/filing.py

```diff
@@ -452,7 +452,11 @@ def ledger(business_id: int,  # pylint: disable=too-many-arguments
     def common_ledger_items(business_identifier: str, filing_storage: FilingStorage) -> dict:
         """Return attributes and links that also get included in T-business filings."""
         no_output_filing_types = ['Involuntary Dissolution', 'conversion']
-        base_url = current_app.config.get('LEGAL_API_BASE_URL')
+
+        # This is a temporary fix so that filingLink, commentLink, and documentLink won't break
+        # Change to use BUSINESS API GW URL below once the Migration to GCP is complete
+        # base_url = current_app.config.get('LEGAL_API_BASE_URL')
+        base_url = current_app.config.get('LEGAL_API_URL')
```

Comment:
so you are only updating this to use `LEGAL_API_URL`.  there are a lot of places in code using `LEGAL_API_BASE_URL`.  Is this ok?

Reply:
@vysakh-menon-aot mentioned the same. Would it be better to update the `LEGAL_API_BASE_URL` itself.

![image](https://github.com/user-attachments/assets/e828ac6f-c343-4127-949a-c6dac621f274)

Reply:
yes, updating `LEGAL_API_BASE_URL` probably makes more sense.  That way it's minimal work to flip it back to the GW url.